crates/ere-dockerized/src/lib.rs

```diff
@@ -161,8 +161,15 @@ impl ErezkVM {
                 )
                 .tag(self.base_zkvm_tag(CRATE_VERSION))
                 .tag(self.base_zkvm_tag("latest"))
-                .bulid_arg("BASE_IMAGE_TAG", self.base_tag(CRATE_VERSION))
-                .exec(&workspace_dir)
+                .bulid_arg("BASE_IMAGE_TAG", self.base_tag(CRATE_VERSION));
```

Comment:
typo: s/bulid_arg/build_arg

Reply:
Fixed in https://github.com/eth-act/ere/pull/113/commits/3340fce0bbf41b70b6dc8805bff9497f15b3f889